modules/serverless-tls-verifying-transport-encryption-eventing.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * /serverless/Eventing/serverless-config-tls-encryption-eventing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-tls-verifying-transport-encryption-eventing_{context}"]
+= Verifying transport encryption in Eventing
+
+To confirm that transport encryption is correctly configured, you can create and test an `InMemoryChannel`. Follow the steps to ensure that the channel uses HTTPS as expected.
```

Comment:
```suggestion
To confirm that transport encryption is correctly configured, you can create and test an `InMemoryChannel` resource. Follow the steps to ensure that the channel uses HTTPS as expected.
```
To clarify what InMemoryChannel is, add a noun after it, such as resource. Also, is `InMemoryChannel` a resource or is it a channel? It looks like it's referred to as both of those terms in this file.

Reply:
It is a resource.